doc/old_and_new_proposal.md

```diff
@@ -108,18 +178,304 @@ addition, every volume gets a "weight" (so far, based on the
+allocate such LVs, the proposal first needs to create (or reuse) a volume
+group that is big enough, which usually means creating one or several physical
+volumes.
+
+In the current implementation, the VG is created to perfectly fit the sizes of
+all the created LVs, with no extra unused space. It would be easy to add an
+configuration option to force the proposal to take more space than strictly
+needed by adding two possibilities: one for using all the available space
+(after deleting partitions according to the settings explained above) and
+another to use a fixed size for the VG (which must be, of course, equal or
+bigger than the sum of the max sizes of all the volumes).
+
+## Proposed settings for yast-storage-ng
+
+The redesign in the approach and code of the storage proposal explained above
+also deserves a revamp of the corresponding section in the `control.xml` file.
+This part of the document presents a possible alternative for the `partitioning`
+section of the control file. The goal is to provide more flexibility defining
+the products behavior and better access to all the features of the new storage
+proposal.
+
+In order to fully understand this section and its implications is important to
+have read and understood the above section titled "How the new proposal
+distributes the space".
+
+### General structure
+
+The new `partitioning` section will still contain the
+`root_subvolume_read_only`, `proposal_settings_editable`,
+`expert_partitioner_warning` and `use_separate_multipath_module` options
+described in this document's subsection "Other general options" at "Settings in
+yast-storage". These options will retain its exact form and meaning.
+
+In addition that, there will be two new subsections: `proposal` and `volumes`.
+
+### The `proposal` subsection
+
+The `proposal` subsection will be used to configure some general aspects of the
+storage proposal (referenced as "guided setup" in the UI) and will contain the
+following options.
+
+  * `default_lvm`. Whether LVM should be used by default.
+  * `default_encrypt`. Whether encryption should be used by default.
+  * `default_windows_delete`. Default value for the automatic delete mode for
+    Windows partitions. It can be `none`, `all` or `ondemand`. For more
+    information, see the description of the new proposal above.
+  * `default_linux_delete`. Default value for the automatic delete mode for
+    Linux partitions. See `default_windows_delete` above.
+  * `default_other_delete`. Default value for the automatic delete mode for
+    other partitions. See `default_windows_delete` above.
+  * `default_resize_windows`. Default value for the user setting deciding
+    whether to resize Windows systems if needed.
+  * `lvm_vg_strategy`. If the user decides to use LVM, strategy to decide the
+    size of the volume group (and, thus, the number and size of created physical
+    volumes). It can be `use_available`, `use_needed` or `use_vg_size`. See the
```

Comment:
For the possible `lvm_vg_strategy` options, I understood the following:

* `use_available`: the VG will be created with all available free space in the system, then the VG size can be greater than the sum of LVs size.
* `use_needed`: the VG size is equal to LVs size.
* `use_vg_size`: fix a size for the VG, so VG size is greater or equal than LVs size.

Am I right? Maybe this explanation should be included.

Reply:
Yes, you are (right). I will include that.